src/Registry.php

```diff
@@ -204,7 +204,7 @@ public function get($path, $default = null)
         }
 
         if ($this->separator === null || $this->separator === '' || !\strpos($path, $this->separator)) {
-            return (isset($this->data->$path) && $this->data->$path !== null && $this->data->$path !== '')
+            return (isset($this->data->$path) && $this->data->$path !== '')
```

Comment:
```suggestion
            return $this->data?->$path !== ''
```
Can't we use the null-safe operator?

Reply:
please don't use this, it's more unreadable then the other syntax. make it more readable and not worse. php is not javascript or perl.